InventoryManagementSystem/Inventory.cs

```diff
@@ -0,0 +1,103 @@
+﻿
+
+using System;
+namespace InventoryManagementSystem
+{
+	public class Inventory
+	{
+		private List<Product> products = new List<Product>();
+		public Inventory()
+		{
+		
+		}
+
+		public void AddProduct(Product product)
+		{
+			products.Add(product);
+			Console.WriteLine("Product added successfully.");
+		}
+
+		public void ViewProducts()
+		{
+			if(products.Count() == 0)
+			{
+				Console.WriteLine("No products.");
+				return;
+			}
+
+			foreach (var product in products)
+			{
+				Console.WriteLine(product.ToString());
+			}
+		}
+		public void EditProduct(string name) 
```

Comment:
Don't you think this function has multiple responsibilities? 
It's reading from the console the values you want to edit, and editing the existing entity, should not we separate these 2 responsibilities? what do you think we can do?